Tests/IndexStoreDBTests/IndexTests.swift

```diff
@@ -254,6 +254,14 @@ final class IndexTests: XCTestCase {
   }
 
   func testOutOfDateEvent() throws {
+    XCTSkipIf(
+      true,
+      """
+      Temporarily disabled due to failing CI: https://ci.swift.org/view/Dashboard/job/oss-swift-package-ubuntu-20_04-aarch64/691/
```

Comment:
That'll get out of date quick, but it shouldn't be disabled for very long so that's probably fine 👍

Reply:
Oh, are you missing a `try` here?